docs/reference/rest-api/security/query-role.asciidoc

```diff
@@ -31,9 +31,13 @@ Retrieves roles with <<query-dsl,Query DSL>> in a <<paginate-search-results,pagi
 
 The role management APIs are generally the preferred way to manage roles, rather than using
 <<roles-management-file,file-based role management>>.
-The query roles API does not retrieve roles that are defined in roles files, nor <<built-in-roles,built-in>> ones.
+The query roles API does not retrieve roles that are defined in `roles.yml` files.
 You can optionally filter the results with a query. Also, the results can be paginated and sorted.
 
+NOTE: As of Elasticsearch version *8.18.0*, this API automatically returns <<built-in-roles,built-in>> roles as well.
```

Comment:
Nit: not sure if we want to mention an ES version here. Wouldn't current docs be >=8.18 already?

Reply:
Good point! I'll remove it.